xml/ssh_banners.xml

```diff
@@ -789,6 +789,43 @@ fingerprint SSH servers.
       <param pos="0" name="os.family" value="Comware"/>
       <param pos="1" name="os.version"/>
    </fingerprint>
+
+   <fingerprint pattern="^(\d\.\d\.\d) SSH Secure Shell OpenVMS V\d\.\d$">
+      <description>SSH for OpenVMS </description>
+      <example service.component.version="3.2.0">3.2.0 SSH Secure Shell OpenVMS V5.5</example>
+      <!--V5.5 refers to TCP/IP Services for OpenVMS version -->		
+      <param pos="1" name="service.component.version"/>
+      <param pos="0" name="service.component.vendor" value="SSH Communication Security"/>
+      <param pos="0" name="service.component.family" value="SSH Secure Shell"/>
+      <param pos="0" name="service.component.product" value="SSH Secure Shell"/>		  
+      <param pos="0" name="service.vendor" value="HP"/>
+      <param pos="0" name="service.family" value="OpenVMS"/>
+      <param pos="0" name="service.product" value="SSH Server"/>
+      <param pos="0" name="os.vendor" value="HP"/>
+      <param pos="0" name="os.device" value="General"/>
+      <param pos="0" name="os.family" value="OpenVMS"/>
+      <param pos="0" name="os.certainty" value="0.75"/>
+   </fingerprint>
+
+   <fingerprint pattern="^(\d\.\d\.\d) SSH (?:Secure Shell )?OpenVMS V\d\.\d VMS_sftp_version (\d)$">
```

Comment:
Here and in the others you've added, it may make sense to make the version regular expressions a little more flexible unless there is good reason not to.  For example, this won't match if the component version is 3.11.0.